assets/filepermissions.yaml

```diff
@@ -53,6 +53,8 @@
 "inbuilt/transformers/dockerfile/javamaven/javamaven.yaml" : 0644
 "inbuilt/transformers/dockerfile/javamaven/m2kdetect.sh" : 0755
 "inbuilt/transformers/dockerfile/javamaven/templates/Dockerfile" : 0644
+"inbuilt/transformers/dockerfile/net5app/net5app.yaml" : 0644
```

Comment:
Can we possibly rename all references to net5 as dotnet5, so that it immediately is obvious to a new person?

Reply:
Yes sure.